### DISCUSSION TITLE
Testing Values used by codewars

### DISCUSSION THREAD

**Discussion by tooMuchRedundantCode** [codewars/codewars.com#3022] [General]
Hi everyone, I'm new to codewars (and coding in general) and had a quick question...

If I'm working on a kata and I pass the initial testing but then fail the 2nd round, is there a way I can see what codewars is using as input values that are causing my code to return an incorrect value? I feel like debugging the code would be easier if I had that data. Thanks!

**Comment by Madjosz** [comment]
You can print/log the input values to the console with `console.log(input)` / `print(input)` or whatever the standard way of printing is in your language. Please note that the logged values will mostly appear above the "Test failed" assertion message of the belonging test and if it's multiple tests in a row then the last logged line will be the one of the failing test.
You might also want to look into the [Codewars Docs: Troubleshooting Your Solution](https://docs.codewars.com/training/troubleshooting/#print-input).

**Reply by Patrickamanyire** [reply]
please help me ont the above code its not passing

**Reply by hobovsky** [reply]
For help with solving Codewars problems please use kata discourse or, even better, `#help-solve` channel of Codewars Gitter.

Github discourses are not meant for help with solving Codewars tasks.
